R/group_apply.R

```diff
@@ -100,13 +100,15 @@ group_apply <- function(occdf, group, fun, ...) {
   }
 
   supp_args <- list(...)
-  indx <- which(!(names(supp_args) %in% names(formals(fun))))
-  if (length(indx) > 1) {
-    stop(paste(paste0("`", names(supp_args)[indx], "`", collapse = "/"),
-               "are not valid arguments for the specified function"))
-  } else if (length(indx) == 1) {
-    stop(paste0("`", names(supp_args)[indx], "`",
-                " is not a valid argument for the specified function"))
+  if (!("..." %in% names(formals(fun)))) {
```

Comment:
The updates with group_apply look good to me and probably something we should have implemented from the start! Nice job!